.clang-tidy

```diff
@@ -10,6 +10,7 @@ Checks: >
   -cppcoreguidelines-avoid-non-const-global-variables,
   -cppcoreguidelines-macro-usage,
   -cppcoreguidelines-pro-bounds-pointer-arithmetic,
+  -cppcoreguidelines-missing-std-forward,
```

Comment:
This is turned off because it is faulty - it doesn't recognize forwarding within a lambda that has captured by reference. Also it is faulty from the point of view that a forwarding reference is not always used to forward...